### ISSUE TITLE
Not recording after update to v.1.1.1

### ISSUE THREAD

**Issue by LeoHald** [HearthSim/Hearthstone-Deck-Tracker#2890]
After downloading v.1.1.1 and upgrading (from 1.1.0) the Deck Tracker stopped recording matches.

**Comment by riQQ** [comment]
Is the overlay working?
If not: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/FAQ#its-not-detecting-cards-i-or-my-opponent-play

**Comment by LeoHald** [comment]
Yes, the overlay is working fine, but Stats - Constructed - Matches does not show new matches.

**Comment by riQQ** [comment]
Have you checked these points: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/FAQ#games-stats?

**Comment by EggyW** [comment]
I have the same problem, I tried all of that, but it appears it simply isn't detecting the game. Also can't import any decks.

**Comment by LeoHald** [comment]
In Options - Tracker - Stats - Record I have the following options checked: Ranked, Arena, Brawl, Casual and Friendly. But Stats - Matches have not recorded anything since the update. I might even be missing one or two matches prior to the update, if my memory serves me right, so the problem might not be directly related to the 1.1.1 upgrade.

**Comment by riQQ** [comment]
Can you post a log please: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/Reporting-Issues?

**Comment by EggyW** [comment]
[hdt_log.txt](https://github.com/HearthSim/Hearthstone-Deck-Tracker/files/531075/hdt_log.txt)

This is what happens when I run both hearthstone and HDT, then click import new deck. It then asks me to enter the play menu, which I did, but nothing happened. Manually creating a deck does not track the cards that are being played either or record the game.

I am running both hearthstone and HDT as admin on windows 10 in windows 7 compatibility mode (trying everything I can think of)

**Comment by riQQ** [comment]
@LeoHald have you tried reinstalling Hearthstone?

**Comment by LeoHald** [comment]
[hdt_log.txt](https://github.com/HearthSim/Hearthstone-Deck-Tracker/files/531841/hdt_log.txt)
Found no errors i Windows Application Log 

I started HDT first, then Battle.net and Hearthstone.
The overlay updated as it should when I chose a new deck.
Played one match. Opponents hand was recorded as it should be during the match, as well as mine.
Shut down Hearthstone again.
In the HDT main window, the deck stats have been updated and shows the correct win rate, but when launching Stats - Constructed - Matches, no new match has been recorded.

Help resolving this is greatly appreciated.
